### PR TITLE
remove helper

### PR DESCRIPTION
**Pull request by raphaelgavache**
Remove helper

### CODE REVIEW

File: pkg/fleet/installer/repository/repository.go [hunk 1]
```diff
@@ -366,16 +367,10 @@ func movePackageFromSource(ctx context.Context, packageName string, rootPath str
 	if err := os.Chmod(targetPath, 0755); err != nil {
 		return "", fmt.Errorf("could not set permissions on package: %w", err)
 	}
-	switch filepath.Base(rootPath) {
-	case "datadog-agent":
-		if err := service.ChownDDAgent(ctx, targetPath); err != nil {
+	if filepath.Base(rootPath) == "datadog-agent" {
+		if err := exec.Command("chown", "-R", "dd-agent:dd-agent", targetPath).Run(); err != nil {
```
Comment (BaptisteFoy) [inline pkg/fleet/installer/repository/repository.go:371]: **💭 Thought:** ‏(non blocking) this should go in postinst

File: pkg/fleet/installer/service/cmd_executor.go
Comment (BaptisteFoy) [inline pkg/fleet/installer/service/cmd_executor.go:1]: lets yeet the whole file

File: pkg/fleet/installer/service/datadog_agent.go [hunk 2]
```diff
@@ -95,7 +96,7 @@ func SetupAgent(ctx context.Context) (err error) {
 			return
 		}
 	}
-	if err = createAgentSymlink(ctx); err != nil {
+	if err = os.Symlink("/opt/datadog-packages/datadog-agent/stable/bin/agent/agent", agentSymlink); err != nil {
```
Comment (BaptisteFoy) [inline pkg/fleet/installer/service/datadog_agent.go:99]: **🔨 Warning:** ‏this isn't a force symlink by default (= if symlink already exists it errors), we can `ln -sf` instead

**Review by arbll — APPROVED** [review]

**Pull request merged**